pkg/test/cluster/environment.go

```diff
@@ -39,3 +46,17 @@ func (e *Environment) GetAPIServer() test.DeployedAPIServer {
 func (e *Environment) GetIstioComponent(k test.DeployedServiceKind) []test.DeployedIstioComponent {
 	return []test.DeployedIstioComponent{nil}
 }
+
+// GetApp implements the test.Environment interface
+func (e *Environment) GetApp(name string) (test.DeployedApp, error) {
+	return getApp(name, e.AppNamespace)
+}
+
+// GetAppOrFail implements the test.Environment interface
+func (e *Environment) GetAppOrFail(name string, t *testing.T) test.DeployedApp {
```

Comment:
Should we use the testing.T that was passed in to the environment originally?

Reply:
I don't think so. This may be called from a subtest of `Environment.T`, in which case we only want to fail the subtest, not the entire suite.